src/vivarium_gates_mncnh/components/mortality.py

```diff
@@ -35,13 +35,19 @@ def configuration_defaults(self) -> dict[str, Any]:
                         )
                         for cause in self.maternal_disorders
                     },
+                    **{
```

Comment:
If you're including disability in this component, it is no longer a "Mortality" component. It might be worth reconsidering its name.